Sources/LanguageServerProtocol/Notifications/SourceKitFileStatusNotification.swift

```diff
@@ -0,0 +1,48 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// Notification from the server about a per-file status update for the client.
```

Comment:
I've been putting `**LSP Extension**` into the first line of the description so it shows up prominently in quick help/hover.

Reply:
Done